sema/subyt/sources.py

```
@@ -408,20 +442,29 @@ def __init__(self, csv_file_path: Path, config: dict = {}) -> None:
             self._csv: Path = csv_file_path.absolute()
             self._init_source(self._csv)
             self._csvconfig: dict = dict()
-            for key in ["delimiter", "quotechar", "header"]:
+            for key in self.CFGKEYS:
                 if key in config:
                     self._csvconfig[key] = config[key]
 
         def __enter__(self) -> object:
             self._csvfile = open(self._csv, mode="r", encoding="utf-8-sig")
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)